app/models/line_item.rb

```diff
@@ -1,5 +1,10 @@
 class LineItem < ApplicationRecord
-    belongs_to :product
-    belongs_to :cart
-    belongs_to :order,optional: true
-end
+  belongs_to :product
+  belongs_to :cart
+  belongs_to :order,optional: true
```

Comment:
Add space before optional: true.

Install rubocop to identify these type of errors

Reply:
installed rubocop